### ISSUE TITLE
Web UI?

### ISSUE THREAD

**Issue by nickwales** [EvanKrall/puppet-vault#4]
Is there a web UI for vault? I haven't seen anything in the documentation or downloadable.

This suggests there is:
https://github.com/EvanKrall/puppet-vault/blob/master/manifests/install.pp#L34-L50

**Comment by ptierno** [comment]
I dont believe there is a vault web ui. I think that piece of puppet code is a remnant from the consul module this module is based off of. The `$vault::ui_` variables dont exist anywhere.
